test/e2e/storage/vsphere/nodemapper.go

```diff
@@ -89,25 +89,23 @@ func (nm *NodeMapper) GenerateNodeMap(vSphereInstances map[string]*VSphere, node
 	}
 
 	for _, node := range nodeList.Items {
-		n := node
```

Comment:
This line and following code in this function , that this PR alters, is needed and shouldn't be altered. Note that value of 'node' will get altered with every iteration, and it's passed to go routine. This will lead to modification of value of closure variable in go routine. That's why 'n' is introduced to shorten the scope and pass node safely to go routine.

 https://github.com/golang/go/wiki/CommonMistakes explains it very well.

Reply:
Fixed.